testbed/tests/test_window.py

```diff
@@ -479,6 +490,35 @@ async def test_full_screen(second_window, second_window_probe):
         assert not second_window_probe.is_full_screen
         assert second_window_probe.content_size == initial_content_size
 
+    # Test the `position`, `screen_position` and `screen`.
+    @pytest.mark.parametrize(
+        "second_window_kwargs",
+        [dict(title="Secondary Window", position=(200, 150))],
+    )
+    async def test_screen(second_window, second_window_probe):
+        """The window can be relocated to another screen, using both absolute and relative screen positions."""
+
+        initial_position = second_window.position
+
+        # Move the window using absolute position.
+        second_window.position = (200, 200)
+        await second_window_probe.wait_for_window("Secondary window has been moved")
+        assert second_window.position != initial_position
+
+        # `position` and `screen_position` will be same as the window will be in primary screen.
+        assert second_window.position == (200, 200)
+        assert second_window.screen_position == (200, 200)
+
+        # Move the window between available screens and assert its `screen_position`
+        for screen in second_window.app.screens:
+            second_window.screen = screen
+            await second_window_probe.wait_for_window("Secondary window has been moved")
```

Comment:
For diagnostic purposes, it would be helpful if this output a *different* message for each screen (either the screen name, or an integer index)

Reply:
Changed it.